podcasts/ThemeableCollectionView.swift

```diff
@@ -26,3 +33,32 @@ class ThemeableCollectionView: UICollectionView {
         indicatorStyle = AppTheme.indicatorStyle()
     }
 }
+
+extension ThemeableCollectionView: AutoScrollCollectionViewDelegate {
+    var timer: Timer? {
```

Comment:
We should move this to `ThemeableCollectionView`, otherwise, `timer` will always be `null`.

Reply:
Why is it not ok for the timer to be nulll? Shouldn't the time be null till initialized from the instance of ThemeableCollectionView?

Reply:
Ah! I thought `Timer.scheduledTimer` initializes it, but its indeed nil. How come it works then? 😬

Reply:
> How come it works then? 😬

I asked myself the same. 😬 I believe since we set `self` as the target it holds a reference to the timer. So it's there, in the memory 👻

Reply:
I moved everything to ThemeableCollectionView and it still returns nil 🤔